README.md

```diff
@@ -739,6 +740,40 @@ More details are in [its own README](./cmd/scanblocks/README.md). The repository
 also includes a shell script, jsonarray2csv.sh, to convert the result into a
 comma-separated value (CSV) file.
 
+## Front End Development
+
+Make sure you have a recent version of [node and npm](https://nodejs.org/en/download/) installed.
+You may want to use the [node version manager (nvm)](https://github.com/creationix/nvm) for managing
+your node download and installation.
+
+From the dcrdata root directory, run the following command to install the node modules.
+
+`npm install`
+
+For development, there's a webpack script that watches for file changes and automatically compiles.
+To use it, run this command in a separate terminal and leave it running while you work.
+
+`npm run watch`
+
+For production, bundle assets via:
```

Comment:
Is it optional?  And I'm wondering the same thing as in my previous commit about where the outputs go.  It seems to be generating public/js/dist/app.bundle.js.  Is that all it makes?

Reply:
Only if you're editing javascript files.